sdk/storage/azure-storage-blobs/test/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.13)
+
+# CodeCoverage will validate requirements
+include(CodeCoverage)
```

Comment:
This won't work from here.
Please try to generate code coverage reports locally to make sure it would still work after moving this here.

Having this here means that the test exe will be compiled with the flags for profiling, like if we were wanting the coverage for the test code.